src/controller/cve.controller/cve.controller.js

```diff
@@ -67,35 +67,48 @@ async function submitCve (req, res, next) {
 }
 
 // Updates a CVE if one exists for the specified CVE ID
-async function updateCve (req, res) {
-  const cve = req.body
-  const cveId = req.params.id
+async function updateCve (req, res, next) {
+  try {
+    const cve = req.ctx.body
+    const cveId = req.ctx.params.id
+    const cveRepo = req.ctx.repositories.getCveRepository()
+    const orgRepo = req.ctx.repositories.getOrgRepository()
 
-  if (cveId !== cve.CVE_data_meta.ID) {
-    return res.status(400).json({ message: cveId + ' parameter does not match ' + cve.CVE_data_meta.ID + ' in the JSON body.' })
-  }
+    // cve id in body must match id in URL param
+    if (cveId !== cve.CVE_data_meta.ID) {
+      return res.status(400).json(error.cveIdMismatch())
+    }
+
```

Comment:
Need to check that the user does not update a CVE to a RESERVED state. CVEs with the RESERVED schema will pass the JSON validator.